cdk/domino_cdk/config/__init__.py

```diff
@@ -0,0 +1,115 @@
+from semantic_version import Version
+
+from domino_cdk import __version__
+from domino_cdk.config.base import DominoCDKConfig
+
+
+def config_loader(c: dict):
+    schema = Version(c.pop("schema", "0.0.0")).truncate()
+    if schema == Version("0.0.1"):
+        return DominoCDKConfig.from_0_0_1(c)
+    elif schema == Version("0.0.0"):
+        return DominoCDKConfig.from_0_0_0(c)
+
+
+def config_template(bastion: bool = False, private_api: bool = False, dev_defaults: bool = False):
+    max_nodegroup_azs = 3
+    destroy_on_destroy = False
+    disk_size = 1000
+    platform_instance_type = "m5.2xlarge"
+    platform_min_size = 3
+    if dev_defaults:
+        max_nodegroup_azs = 1
+        destroy_on_destroy = True
+        disk_size = 100
+        platform_instance_type = "m5.4xlarge"
+        platform_min_size = 1
+    return config_loader(
+        {
```

Comment:
I realize it'll fail fast, but constructing objects for this inline makes me feel more comfortable (like the one in the unit test?)

Reply:
Yes, I'll be doing something with this in follow up.